src/oop/object_interactions/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,48 @@
+# oop-object_interactions-medium-q3
+
+- Java question
+
+## Newspaper Class
+
+The `Newspaper` class currently has the following items:
+
+- Attributes:
+    - An `int` called **myYear**
+    - An `int` called **myMonth**
+    - An `int` called **myDay**
+    - A `String` called **myStory**
+    - None of these attributes should be accessible from other classes.
+
+- Constructors:
+    - A constructor with no parameters that initializes all the int attributes to -1 and the **myStory** to an
+      empty `String`.
+    - It is accessible from other classes.
+
+- Methods:
+    - Getter methods for the **myStory**, **myYear**, **myMonth**, and **myDay** attributes that are accessible from other
+      classes.
+
+You must add the following items to the class:
+
+- Constructors:
+    - A constructor that takes three `int` and one `String` in that order and initialized the **myYear**, **myMonth**,
+      **myDay**, and **myStory** attribute in that order.
+    - It should be accessible from other classes.
+
+- Methods:
+    - A `makeCopy` method that take another **Newspaper** object and change the attribute of the object to the attributes
+      of the input parameter **Newspaper** object.
+    - A `toString` method which returns the information about the class in the following `String` structure:
```

Comment:
I also think this toString description is a bit unclear. Instead of "in the order", maybe say "has the values" and specify that you should form the date with slashes between the format YEAR/MONTH/DAY